nltk/translate/util.py

```diff
@@ -0,0 +1,144 @@
+''' UTILITY Functions for ROUGE score for Machine Translation'''
+
+from __future__ import division
+
+def jacknifing(score_list, averaging=True):
+        ''' This is a averaging function for calculating ROUGE score
+    when multiple references are present. In this technique, from a 
+    given list of m values, m unique sets of m-1 elements is created.
+    Then the maximum is calculated from each of this m sets. The 
+    averaging is then performed on the m maximum values thus obtained.
+    This technique has been referred to as the Jacknifing technique 
+    in the original paper that introduces ROUGE-SCORE.
+
+
+    param score_list : list of scores over which averaging occurs
+    type (score_list) : list
+
+    param averaging : Jacknifing occurs if averaging is True
+    type (averaging) : boolean
+
+    >>> scores=[10, 20, 30, 40]
+    
+    >>> jacknifing(scores, averaging=False)
+    [10, 20, 30, 40]
+
+    >>> jacknifing(scores, averaging=True)
+    37.5
+
+    >>> scores =[10]
+    
+    >>> jacknifing(scores)
+    10.0
+
+
+    '''
```

Comment:
Please follow the NLTK style docstring for functions, use triple quotes to open and close and whenever possible put in the parameter description and types for the functions.